store/mockstore/mocktikv/executor.go

```diff
@@ -72,12 +73,17 @@ func (e *tableScanExec) GetSrcExec() executor {
 	return e.src
 }
 
-func (e *tableScanExec) ResetCount() {
-	e.count = 0
+func (e *tableScanExec) ResetCounts() {
+	e.counts[e.start] = 0
```

Comment:
Why only reset one range?
But the name indicates resetting all ranges.

Reply:
The effect is the same as the reset all ranges because ranges before `start` will not be used anymore.